### PR TITLE
Fixed broken syntax on UI_BP.tsv LINE 519.

### LINKED ISSUE #156: Broken syntax on file UI_BP.tsv (LINE 519)
Na linha 519 do arquivo UI_BP.tsv foi usado uma aspa dupla ao invés da aspa simples que tinha no arquivo original.

### PR DESCRIPTION
**Pull request by lmmfranco**
Fixes issue #156

### CODE REVIEW

File: UI_BP.tsv [hunk 1]
```diff
@@ -516,7 +516,7 @@ UI_20150317_000515	{@st41}Excluir tudo
...
-UI_20150317_000519	{@st59b}Quests{nl}{@st59s} - Quando você estiver numa nova quest, ela será adicionada ao registro de quests. {nl} - As quests principais aparecerão em amarelo, enquanto uma subquest aparecerá em azul. {nl} - A seta nos pés do personagem direcionará para o local da quest. {nl} - Se você completar uma quest, um ícone "Retonar" irá aparecer no registro de quests. {nl} (Teleporta você para o NPC da quest através da tecla Backspace ou de um clique do mouse) {nl} - Adicione ou remova quests no registro de quest marcando ou desmarcando a caixa na frente do nome da quest. {/}
+UI_20150317_000519	{@st59b}Quests{nl}{@st59s} - Quando você estiver numa nova quest, ela será adicionada ao registro de quests. {nl} - As quests principais aparecerão em amarelo, enquanto uma subquest aparecerá em azul. {nl} - A seta nos pés do personagem direcionará para o local da quest. {nl} - Se você completar uma quest, um ícone 'Retonar' irá aparecer no registro de quests. {nl} (Teleporta você para o NPC da quest através da tecla Backspace ou de um clique do mouse) {nl} - Adicione ou remova quests no registro de quest marcando ou desmarcando a caixa na frente do nome da quest. {/}
```
Comment (lmmfranco) [inline UI_BP.tsv]: "Retornar" ->  'Retornar'

File: UI_BP.tsv [hunk 2]
```diff
@@ -516,7 +516,7 @@ UI_20150317_000515	{@st41}Excluir tudo
...
-UI_20150317_000519	{@st59b}Quests{nl}{@st59s} - Quando você estiver numa nova quest, ela será adicionada ao registro de quests. {nl} - As quests principais aparecerão em amarelo, enquanto uma subquest aparecerá em azul. {nl} - A seta nos pés do personagem direcionará para o local da quest. {nl} - Se você completar uma quest, um ícone "Retonar" irá aparecer no registro de quests. {nl} (Teleporta você para o NPC da quest através da tecla Backspace ou de um clique do mouse) {nl} - Adicione ou remova quests no registro de quest marcando ou desmarcando a caixa na frente do nome da quest. {/}
+UI_20150317_000519	{@st59b}Quests{nl}{@st59s} - Quando você estiver numa nova quest, ela será adicionada ao registro de quests. {nl} - As quests principais aparecerão em amarelo, enquanto uma subquest aparecerá em azul. {nl} - A seta nos pés do personagem direcionará para o local da quest. {nl} - Se você completar uma quest, um ícone 'Retornar' irá aparecer no registro de quests. {nl} (Teleporta você para o NPC da quest através da tecla Backspace ou de um clique do mouse) {nl} - Adicione ou remova quests no registro de quest marcando ou desmarcando a caixa na frente do nome da quest. {/}
```
Comment (lmmfranco) [inline UI_BP.tsv]: ```"Retonar"``` -> ```'Retornar'```

**Comment by lmmfranco** [comment]
Good to merge

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**